packages/web-react/src/hooks/useScrollControl.ts

```diff
@@ -25,7 +25,7 @@ export const useScrollControl = (ref: MutableRefObject<HTMLDialogElement | null>
     if (isOpen) {
       disableScroll();
     } else if (ref.current && !ref.current.open) {
-      const offsetY = parseFloat(document.body.style.top || '0');
+      const offsetY = parseFloat(document.body.style.top || '0') || window.scrollY;
```

Comment:
I am afraid that this solution is still not what we need in Jobs. 😞 

We need to prevent the page from bouncing once the modal is closed. But we have to consider a scenario where the modal has not been opened yet  – that case falls in to this branch of the condition. That's why when it comes to the native scroll to the anchor in Jobs, we face the problem that only on Firefox we scroll to the top of the page, because Chrome scrolls to the anchor after the execution of this script.

A suggestion for modification is as follows:

```suggestion
      const enableScroll = (offsetY: number) => {
        …
        window.scrollTo(0, offsetY);
      };

      const offsetY = window.scrollY ? window.scrollY : Math.max(-1 * parseFloat(document.body.style.top) || 0, 0);
```

- When we open the modal, setting `position: fixed;` to `body` resets `window.scrollY` to `0`. Hence, we need to ask for the value stored in `top`, which has saved the size of the page's offset scroll. This value corresponds to a downward shift of the page, so it is negative and we need to convert it to positive by multiplying by -1 to get the `y` page coordinate where we want to scroll.
- If the modal is not open, the `top` property is not defined, so we need to look at `window.scrollY` if there was a scroll on the page to prevent an unwanted scrolling to `y: 0`.
- `Math.max` is used to avoid the value of the variable `-0`.

FYI: The code suggestion has been tested on Jobs and it looks like the scrolling and open/close modal works as it should. 🎉

Reply:
@crishpeen @literat What do you think? Can we apply this suggestion or do you see any side effects or improvement ideas?